src/2d/shallow/src2.f90

```diff
@@ -18,6 +27,17 @@ subroutine src2(meqn,mbc,mx,my,xlower,ylower,dx,dy,q,maux,aux,t,dt)
     ! Locals
     integer :: i, j, nman
     real(kind=8) :: h, hu, hv, gamma, dgamma, y, fdt, a(2,2), coeff
+    real(kind=8) :: xm, xc, xp, ym, yc, yp, dx_meters, dy_meters
+    real(kind=8) :: u, v, hu0, hv0
+    real(kind=8) :: tau, wind_speed, theta, phi, psi, P_gradient(2), S(2)
+    real(kind=8) :: Ddt, sloc(2)
+
+    ! Physics
+    real(kind=8), parameter :: fric_coefficient = 7.d0 / 3.d0
```

Comment:
Should this parameter `fric_coefficient` be used in line 69 instead of `7.d0/3.d0`?